shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/presentation/module/HomeViewModel.kt

```diff
@@ -18,56 +14,78 @@ private const val DEFAULT_PAGE_SIZE = 5
 data class HomeViewState(
     val isLoading: Boolean,
     val headerUiModel: SurveyHeaderUiModel? = null,
-    var surveys: List<SurveyUiModel> = listOf()
+    var surveys: List<SurveyUiModel> = listOf(),
 ) {
-    constructor() : this (true)
+    constructor() : this(true)
 }
 
 class HomeViewModel(
     private val getUserProfileUseCase: GetUserProfileUseCase,
     private val getSurveysUseCase: GetSurveysUseCase,
+    private val getAppVersionUseCase: GetAppVersionUseCase,
     private val dateTime: DateTime,
     private val dateTimeFormatter: DateTimeFormatter,
 ) : BaseViewModel() {
 
     private val _viewState: MutableStateFlow<HomeViewState> = MutableStateFlow(HomeViewState())
-
     val viewState: StateFlow<HomeViewState> = _viewState
 
+    private val _appVersion: MutableStateFlow<String> = MutableStateFlow("")
+    val appVersion: StateFlow<String> = _appVersion
+
     private var previousSelectedIndex = 0
     private var currentPage = 1
 
+    init {
+        fetchData()
+    }
+
     fun fetchData() {
```

Comment:
```suggestion

    private fun fetchData() {
```

Reply:
@kaungkhantsoe please check [this](https://github.com/nimblehq/avishek-phong-kmm-ic/pull/105#discussion_r1194756576) comment.